web/components/DateRange/DateRange.tsx

```diff
@@ -0,0 +1,20 @@
+import { formatDateRange } from '../../util/date';
+
+interface DateRangeProps {
+  startDate: string;
+  endDate: string;
+}
+
+export const DateRange = ({ startDate, endDate }: DateRangeProps) => {
+  if (!startDate || !endDate) {
+    return null;
+  }
+
+  const { start, end } = formatDateRange(startDate, endDate);
```

Comment:
Nit: Maybe call it something like `formatDateRangeInUtc` for clarity?

Reply:
Fixed in c234936.